utils/update_authors.py

```diff
@@ -63,10 +63,10 @@
         if translators['translator-list'] != 'translator-credits':
             locale_message += ' translators and'
             lua_translators += '\t{\n'  # entry
-            lua_translators += f'\t\theading = "{ translators["your-language-name"] }'
+            lua_translators += f'\t\theading = "{ translators['your-language-name'] }'
```

Comment:
This automatic formatting (and all others in this commit) break the code. (Pulled it locally, running the script reports an error on this line.)
How can I prevent this?

Reply:
Looks like a bug in the formatter. I don't know python much, so I may be wrong, but you could try:

`f'''..."{ t['i'] }'''`

Reply:
The try seems to be successful. Thanks.